infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/InfinispanRemoteProperties.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispanremote;
+
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.ogm.datastore.keyvalue.cfg.KeyValueStoreProperties;
+
+/**
+ * Properties for configuring the Infinispan Remote datastore via {@code persistence.xml} or
+ * {@link StandardServiceRegistryBuilder}.
+ */
+public final class InfinispanRemoteProperties implements KeyValueStoreProperties {
+
+	/**
+	 * The configuration property to use as key to define a custom configuration resource
+	 * for the Hot Rod (Infinispan remote) client.
+	 */
+	public static final String CONFIGURATION_RESOURCE_NAME = "hibernate.ogm.infinispan_remote.configuration_resource_name";
```

Comment:
It's not an XML file but a properties file.
This file is mandatory to configure the Hot Rod client.

E.g. this is the one we use for testing:
 - https://github.com/Sanne/hibernate-ogm/blob/5f93b5bcf8974c412233ff260e817dfaa80d0062/infinispan-remote/src/test/resources/hotrod-client-testingconfiguration.properties

We could read such properties from the Hibernate properties? Last time I asked, you suggested to keep the external resource.

There's a follow up JIRA open to strictly validate the properties define in there, as we can't deal with some options being set.